src/main/java/org/softc/armoryexpansion/common/integration/modsupport/ConArmIntegration.java

```diff
@@ -0,0 +1,208 @@
+package org.softc.armoryexpansion.common.integration.modsupport;
+
+import c4.conarm.ConstructsArmory;
+import c4.conarm.lib.materials.ArmorMaterialType;
+import c4.conarm.lib.materials.CoreMaterialStats;
+import c4.conarm.lib.materials.PlatesMaterialStats;
+import c4.conarm.lib.materials.TrimMaterialStats;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import net.minecraft.item.Item;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.EventPriority;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import org.softc.armoryexpansion.ArmoryExpansion;
+import org.softc.armoryexpansion.client.integration.aelib.plugins.tinkers_construct.material.MaterialRenderType;
+import org.softc.armoryexpansion.common.integration.aelib.integration.JsonIntegration;
+import org.softc.armoryexpansion.common.integration.aelib.plugins.constructsarmory.material.ArmorMaterial;
+import org.softc.armoryexpansion.common.integration.aelib.plugins.constructsarmory.material.IArmorMaterial;
+import org.softc.armoryexpansion.common.integration.aelib.plugins.general.material.IBasicMaterial;
+import org.softc.armoryexpansion.common.util.Math;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.library.materials.*;
+import slimeknights.tconstruct.tools.TinkerMaterials;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.Objects;
+
+@Mod(
+        modid = ConArmIntegration.MODID,
+        name = ConArmIntegration.NAME,
+        version = ArmoryExpansion.VERSION,
+        dependencies = ConArmIntegration.DEPENDENCIES
+)
+@Mod.EventBusSubscriber
+public class ConArmIntegration extends JsonIntegration {
+    static final String MODID = ArmoryExpansion.MODID + "-" + ConstructsArmory.MODID;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896610)